drivers/spi/Kconfig.xec_qmspi_v2_ldma

```diff
@@ -0,0 +1,18 @@
+# Microchip XEC QMSPI V2, QMSPI LDMA
+
+# Copyright (c) 2022 Microchip Technology Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+choice
+	prompt "Select Microchip XEC QMSPI driver"
+	default SPI_XEC_QMSPI_LDMA
+	depends on SPI && SOC_SERIES_MEC172X
+	help
+	 Select Microchip XEC QMSPI driver implementation.
+
+config SPI_XEC_QMSPI_LDMA
+	bool "Use QMSPI driver with LDMA (local dma)"
+
+config SPI_XEC_QMSPI_V2
```

Comment:
where is v1? why would someone want to use v1 instead of v2?

should we be deleting v1?

Reply:
v1 is the ldma (local dma based) qmspi driver, we are not deleting it. 
The v2 driver supports only full duplex to work with the Zephyr NOR driver.

Reply:
should it not be called v2 then? based on the name, it sounds like v2 replaces v1, but you say it does not?

Reply:
it replaces it only for a particular use case - to work with Zephyr NOR driver, because the ldma qmspi driver is incompatible with Zephyr NOR.
do you think we should call it differently?

Reply:
we are thinking to change to to `SPI_XEC_QMSPI_SIMPLE` (or `SPI_XEC_QMSPI_FD`) as it does only full duplex. Can you help to confirm?

Reply:
`SPI_XEC_QMSPI_FULL_DUPLEX` would avoid uncommon abbreviations.

Reply:
Thanks!

Reply:
updated to use SPI_XEC_QMSPI_FULL_DUPLEX